### PR TITLE
add new free-programming-interactive-tutorials in ru

### PR DESCRIPTION
**Pull request by shapito27**
add Go, Solidity, Microsoft azure tutorials in Russian language

## What does this PR do?
Add resource(s)

## For resources
free-programming-interactive-tutorials-ru
### Description
add Go, Solidity, Microsoft azure tutorials in Russian language
### Why is this valuable (or not)?
it's popular languages and technologies 
### How do we know it's really free?
it's a well know tutuorials
### For book lists, is it a book? For course lists, is it a course? etc.
it's a interactive courses
## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-interactive-tutorials-ru.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 ### Index
 
 * [Веб-разработка](#Веб-разработка)
```
Comment (eshellman) [inline more/free-programming-interactive-tutorials-ru.md:3]: If you look at the static version of the site (https://ebookfoundation.github.io/free-programming-books/more/free-programming-interactive-tutorials-ru.html) you'll see that the id's are all lower case; which means that the links don't work: `<h3 id="веб-разработка">Веб-разработка</h3>` Change all href fragments to lower case.

File: more/free-programming-interactive-tutorials-ru.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
+* [Облачные вычисления](#Облачные-вычисления)
 * [Git](#git)
+* [Go](#go)
 * [Python](#Python)
```
Comment (eshellman) [inline more/free-programming-interactive-tutorials-ru.md:7]: lower case

File: more/free-programming-interactive-tutorials-ru.md [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
+* [Solidity](#Solidity)
```
Comment (eshellman) [inline more/free-programming-interactive-tutorials-ru.md:8]: lower case

File: more/free-programming-interactive-tutorials-ru.md [hunk 4]
```diff
@@ -12,17 +15,31 @@
...
+* [Первые шаги Go](https://docs.microsoft.com/ru-ru/learn/paths/go-first-steps/)
+
 ### Python
 
 * [Pythontutor](https://pythontutor.ru)
 
 
+### Solidity
+
+* [CryptoZombies.io](https://cryptozombies.io/ru/) - Ethereum DApps
```
[the remark this answers is not in the document]
Comment (eshellman) [reply more/free-programming-interactive-tutorials-ru.md:40]: it's fine like this, but remove trailing slash

File: more/free-programming-interactive-tutorials-ru.md [hunk 5]
```diff
@@ -1,8 +1,11 @@
 ### Index
 
 * [Веб-разработка](#Веб-разработка)
+* [Облачные вычисления](#Облачные-вычисления)
```
Comment (eshellman) [inline more/free-programming-interactive-tutorials-ru.md:4]: lower case

File: more/free-programming-interactive-tutorials-ru.md [hunk 6]
```diff
@@ -1,8 +1,11 @@
...
 * [Git](#git)
+* [Go](#go)
 * [Python](#Python)
+* [Solidity](#Solidity)
 * [SQL](#SQL)
```
Comment (eshellman) [inline more/free-programming-interactive-tutorials-ru.md:9]: ` [SQL](#sql)`

**Review by eshellman — CHANGES_REQUESTED** [review]

**Comment by eshellman** [comment]
This content could easily be made into a book, and I encourage you to do so! Currently it's more of a blog, which is out of scope for this list.

**Comment by shapito27** [comment]
Hi, shall I create a new pull request? It seems, that I can't add changes to the current pull request because it has closed.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books! Sorry for the mistake in closing the PR - I had two PRs mixed up!

**Pull request merged**